packages/terra-list/src/terra-dev-site/doc/example/ListItem.jsx

```diff
@@ -10,12 +10,16 @@ const ListSectionExample = () => (
   <List role="listbox" aria-label="example-label">
```

Comment:
I think we might need to consider adding aria-label as prop for List similar to code changes that are made to fix axe violation in [Infinite-List] (https://github.com/cerner/terra-framework/pull/999).

Reply:
In list aria-label can be passed by custom-prop to the `ul` element so no need of an extra prop.